assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -22,71 +22,92 @@ import placeholderShippingMethods from '../../placeholder-shipping-methods';
 import './style.scss';
 
 /**
- * Component that renders the Cart block when user has something in cart aka "full".
+ * Given an API response with cart totals, generates an array of rows to display in the Cart block.
+ *
+ * @param {Object} cartTotals - Cart totals data as provided by the API.
+ * @returns {Object[]} Values to display in the cart block.
  */
-const Cart = () => {
-	// @todo this are placeholders
-	const onActivateCoupon = ( couponCode ) => {
-		// eslint-disable-next-line no-console
-		console.log( 'coupon activated: ' + couponCode );
-	};
-	const cartTotals = {
-		currency: 'EUR',
-		currency_minor_unit: 2,
-		total_items: 6000,
-		total_items_tax: 0,
-		total_fees: 0,
-		total_fees_tax: 0,
-		total_discount: 0,
-		total_discount_tax: 0,
-		total_shipping: 0,
-		total_shipping_tax: 0,
-		total_tax: 0,
-		total_price: 6000,
-	};
-
-	const currency = getCurrencyFromPriceResponse( cartTotals );
-	const totalItems = [
+const getTotalRows = ( cartTotals ) => {
+	const totalItems = parseInt( cartTotals.total_items );
+	const totalItemsTax = parseInt( cartTotals.total_items_tax );
+	const totalRows = [
 		{
 			label: __( 'List items:', 'woo-gutenberg-products-block' ),
 			value: DISPLAY_PRICES_INCLUDING_TAXES
-				? cartTotals.total_items + cartTotals.total_items_tax
-				: cartTotals.total_items,
+				? totalItems + totalItemsTax
+				: totalItems,
 		},
 	];
-	if ( cartTotals.total_fees > 0 ) {
-		totalItems.push( {
+	const totalFees = parseInt( cartTotals.total_fees );
+	if ( totalFees > 0 ) {
+		const totalFeesTax = parseInt( cartTotals.total_fees_tax );
+		totalRows.push( {
 			label: __( 'Fees:', 'woo-gutenberg-products-block' ),
 			value: DISPLAY_PRICES_INCLUDING_TAXES
-				? cartTotals.total_fees + cartTotals.total_fees_tax
-				: cartTotals.total_fees,
+				? totalFees + totalFeesTax
+				: totalFees,
 		} );
 	}
-	if ( cartTotals.total_discount > 0 ) {
-		totalItems.push( {
+	const totalDiscount = parseInt( cartTotals.total_discount );
+	if ( parseInt( totalDiscount ) > 0 ) {
```

Comment:
Does the `parseInt` need to be used here again when `totalDiscount` is already cast to int?

Reply:
Absolutely not! Removed in 7b6dcac58175130f780080340aa4e6628e1921d0.